RecoParticleFlow/PFClusterProducer/plugins/PFRecHitProducer.cc

```diff
@@ -55,6 +56,10 @@ void
    out->reserve(localRA1.upper());
    cleaned->reserve(localRA2.upper());
    for( const auto& creator : creators_ ) {
+     if (!init_) { // should go in beginRun
+       creator->init(iSetup);
```

Comment:
can everything relevant to this creator be safely defined by a single call per job?
Is some values change with IOVs with transitions possible in the same dataset, some other mechanism is needed to initialize.